### PR TITLE
[Doc] Add 2.5 BE config items

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Review by amber-create — COMMENTED** [review]
image
1. The time -> The amount/length of time
2. 整句改为 -> The minimum time interval at which Cumulative Compaction retries upon failures.
3. “level number" 容易理解为 level 编号；建议改为 -> The number of levels
4. "two contiguous level" 中，level 应该是复数 levels

File: docs/administration/Configuration.md [hunk 1]
```diff
@@ -446,6 +451,12 @@ BE static parameters are as follows.
...
 | block_cache_disk_size  | 0 | Bytes | The maximum amount of data that can be cached in a single disk. For example, if you configure two disk paths for the `block_cache_disk_path` parameter and set the value of the `block_cache_disk_size` parameter as `21474836480` (20 GB), a maximum of 40 GB data can be cached in these two disks. The default value is `0`, which indicates that only the memory is used to cache data. Unit: bytes. |
+| jdbc_connection_pool_size  | 8 | The JDBC connection pool size. Each BE node shares the same connection pool when accessing the external table with the same `jdbc_url`. |
+| jdbc_minimum_idle_connections  | 1 | The minimum number of idle connections in the JDBC connection pool. |
+| jdbc_connection_idle_timeout_ms  | 600000 | The JDBC idle connection timeout. If the connection idle time in the JDBC connection pool exceeds this value, the connection pool closes idle connections of more than the number specified in the configuration item `jdbc_minimum_idle_connections`. |
+| query_cache_capacity  | 536870912 | The size of the Query Cache in BE. |
```
Comment (evelynzhaojie) [inline docs/administration/Configuration.md:457]: 刚查了一下，query cache 特性文档里有这个参数，辛苦立晨根据那个文档里更一下：The size of the query cache in BE. Unit: bytes. The default size is 512 MB. The size cannot be less than 4 MB. If the memory capacity of the BE is insufficient to provision your expected query cache size, you can increase the memory capacity of the BE.

**Review by amber-create — COMMENTED** [review]
image

1. 这句话含义应该不太对。英文的含义是，”BE 之间共用同一个连接池”，实际上，应该是指“一个 BE 上（访问 jdbc_url 相同的外表的）不同 query”吧？
2. "The JDB idle connection timeout" 建议改为 -> The amount/length of time after which an idle connection in the JDBC connection pool. 或者 The timeout period per idle connection in the JDBC connection pool.
3. "the Query Cache" 可以小写 -> "the query cache"；"BE” 前面加冠词 the（特指当前配置参数中的 BE）

File: docs/administration/Configuration.md [hunk 2]
```diff
@@ -446,6 +451,12 @@ BE static parameters are as follows.
 | block_cache_block_size | 1048576  | Bytes | The size of each block. Unit: bytes. The default value is `1048576`, which is 1 MB. |
 | block_cache_mem_size   | 2147483648 | Bytes | The maximum amount of data that can be cached in the memory. Unit: bytes. The default value is `2147483648`, which is 2 GB. We recommend that you set the value of this parameter to at least 20 GB. If StarRocks reads a large amount of data from disks after block cache is enabled, consider increasing the value. |
 | block_cache_disk_size  | 0 | Bytes | The maximum amount of data that can be cached in a single disk. For example, if you configure two disk paths for the `block_cache_disk_path` parameter and set the value of the `block_cache_disk_size` parameter as `21474836480` (20 GB), a maximum of 40 GB data can be cached in these two disks. The default value is `0`, which indicates that only the memory is used to cache data. Unit: bytes. |
+| jdbc_connection_pool_size  | 8 | The JDBC connection pool size. Each BE node shares the same connection pool when accessing the external table with the same `jdbc_url`. |
```
Comment (evelynzhaojie) [inline docs/administration/Configuration.md:454]: Each BE node share 这个表达不太通顺，一般主语至少两个东西才会去 share。建议改为 BEs share the same connection pool when accessing an external table xxx

**Review by amber-create — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by EsoragotoSpirit** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**